src/creep.prototype.js

```diff
@@ -99,7 +103,11 @@ function positionAtDirection(origin, direction) {
   if (x > 49 || x < 0 || y > 49 || y < 0) {
     return;
   }
-  return new RoomPosition(x, y, origin.roomName);
+  try {
+    return new RoomPosition(x, y, origin.roomName);
+  } catch (e) {
+    console.log('Failed to set position', x, y, origin.roomName, e);
```

Comment:
This threw an exception for me at some point, but was probably due to an upstream bug that has since been resolved.  Added this here for protection